### PR TITLE
StopFinder options

### PR DESCRIPTION
**Pull request by danji90**
- Added options parameter to StopFinder
- These are converted to a string of URL query parameters

### CODE REVIEW

File: src/components/Search/engines/StopFinder.js [hunk 1]
```diff
@@ -3,13 +3,19 @@ import React from 'react';
 import Engine from './Engine';
 
 class StopFinder extends Engine {
-  constructor(endpoint = 'https://api.geops.io/stops/v1/') {
+  constructor(endpoint = 'https://api.geops.io/stops/v1/', options) {
```
Comment (friedjoff) [inline src/components/Search/engines/StopFinder.js]:
```suggestion
  constructor(endpoint = 'https://api.geops.io/stops/v1/', options = {}) {
```

File: src/components/Search/engines/StopFinder.js [hunk 2]
```diff
@@ -3,13 +3,19 @@ import React from 'react';
...
     super();
+    this.params = options || {};
```
Comment (friedjoff) [inline src/components/Search/engines/StopFinder.js]:
```suggestion
    this.options = options;
```

Renaming the function parameter seems confusing to me.

File: src/components/Search/engines/StopFinder.js [hunk 3]
```diff
@@ -3,13 +3,19 @@ import React from 'react';
...
     this.endpoint = endpoint;
   }
 
   search(value) {
-    return fetch(`${this.endpoint}?&q=${value}&key=${this.apiKey}`)
+    const paramString = `${Object.keys(this.params)
```
Comment (friedjoff) [inline src/components/Search/engines/StopFinder.js]: I would prefer to extract valid API parameters from `options` explicitly to make it easier to understand whats going on.

Comment (danji90) [reply src/components/Search/engines/StopFinder.js]: Agreed, thanks for pointing that out

**Pull request merged**